app/views/articles/edit.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Edit Article</h1>
+<!-- Page Header -->
+<header class="masthead" style="background-image: url(<%= asset_path 'create-bg.jpg' %>);">
+  <div class="overlay"></div>
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-8 col-md-10 mx-auto">
+        <div class="page-heading">
+          <h1>Edit Article</h1>
+          <span class="subheading">¡Let's enhance this post!</span>
+        </div>
+      </div>
+    </div>
+  </div>
+</header>
 <%= render 'form', article: @article %>
+
```

Comment:
Please delete the extra line at the end of the file.

Reply:
Done!